### ISSUE TITLE
a11y: better visually defined X when placed on colored backgrounds

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-dialog#60]
### Feature Description

The current auro-dialog has a simple UI update when used with dark background images. Using the `ondark` option will update the `X` to appear white versus black. While this may work in simple situations where the majority of the background is dark, it will not work with complex colored backgrounds. 

The issue is more complicated with resizing the viewport, which allows the `X` to travel over the area of an image, creating accessibility issues. 

![Image](https://github.com/user-attachments/assets/edaa7668-bf1f-4d58-a053-700aab3e94a3)

### Relevant WCAG 2.2 Criteria:

* **Success Criterion 1.4.11 – Non-text Contrast**
  *"The visual presentation of user interface components and graphical objects must have a contrast ratio of at least 3:1 against adjacent colors."*

This fails to meet WCAG 2.2 standards and poses a usability and accessibility barrier.

### Recommended feature:

To meet accessibility requirements and improve visibility, the "X" icon should use a background shape (e.g., circle or box) to provide sufficient contrast against the surrounding area. 

The following edited image example would pass Success Criterion 1.4.11 – Non-text Contrast.

![Image](https://github.com/user-attachments/assets/e6e47edc-c251-487c-99e2-98a942d95e0f)

### Additional context

See ADO https://itsals.visualstudio.com/E_Sell_Merchandising/_workitems/edit/1247567

**Comment by blackfalcon** [comment]
From what I can see, the design of the dialog boxes has changed considerably so this is no longer an issue.
